setup.py

```diff
@@ -105,6 +114,16 @@ def build_extension(self, ext):
         else:
             super().build_extension(ext)
 
+    def initialize_options(self):
+        super().initialize_options()
+        self.antlr_runtime = 'static'
+
+    def finalize_options(self):
+        super().finalize_options()
+        assert self.antlr_runtime in [
+            'static', 'shared'
+        ], 'Invalid antlr_runtime'
```

Comment:
Change message to `'Invalid antr_runtime {}, expected one of {}'.format(self.antr_runtime, ANTLR_RUNTIMES)`, etc?

Reply:
Done.